functions/register.go

```diff
@@ -28,6 +29,9 @@ func Register(token string, isGui bool) error {
 	if err = json.Unmarshal(data, &serverData); err != nil {
 		logger.FatalLog("could not read enrollment token")
 	}
+	if config.CurrServer != "" && config.CurrServer != strings.TrimSpace(serverData.Server[4:len(serverData.Server)]) {
+		fmt.Println("WARNING: Joining any network on another server will disconnect netclient from the networks of the current server ->", config.CurrServer)
+	}
```

Comment:
you can do this after the register response, you will have the server name in the response, you can avoid slicing the data

Reply:
done